test/images/.permitted-images

```diff
@@ -0,0 +1,50 @@
+# NOTE: Please do NOT add any to this list!!
```

Comment:
Should this really be a dot file and thus hidden by default when listing the directory in a shell?

I've seen that done elsewhere, but I can't think of a good reason for it. Am I missing something? :thinking:

Reply:
It's data that most users should not be editing.

Reply:
Yeah, I mean the path is clearly referenced in the script so owners should be able to find it, anyone else should not touch it anyhow.

Reply:
It's a convention we have, I think if we want to revisit that it should be a follow-up.